src/chain_spec.rs

```diff
@@ -28,18 +28,18 @@ use std::iter;
 
 const STAGING_TELEMETRY_URL: &str = "wss://telemetry.polymesh.live/submit/";
```

Comment:
Let's change this

Reply:
@cwsatpolymath suggested reusing the same end point for V2 and using tags to distinguish the networks. I'm agnostic either way.

Reply:
Sounds fine to me. Substrate telemetry supports multiple networks natively anyway.